pkgs/servers/http/apache-modules/mod_ca/default.nix

```diff
@@ -1,16 +1,16 @@
-{ stdenv, fetchurl, pkgconfig, apacheHttpd, openssl, openldap }:
+{ stdenv, fetchurl, apacheHttpd, pkgs, pkgconfig }:
```

Comment:
Please include packages directly instead of referencing via `pkgs.openssl`, for example. Same goes for `apr`, `aprutil`, and `openldap`.

Reply:
Ah sorry - so I went the wrong direction ? I noticed other recent packages doing exactly this; and thought this was the new norm to simplify things. 

Will revert back & make it explicit. Apologies.

Reply:
Should be fixed.

Reply:
Modules use `pkgs` and this might be where you saw it. Generally packages should include each package explicitly.